src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2586,7 +2586,7 @@ class ManagedAppDefinitionScenarioTest(ScenarioTest):
     def test_managedappdef(self, resource_group):
 
         self.kwargs.update({
-            'upn': self.create_random_name('testuser', 15) + '@azuresdkteam.onmicrosoft.com',
+            'upn': self.create_random_name('testuser', 15) + '@detienne001gmail.onmicrosoft.com',
```

Comment:
Could you not change this email address to personal email address? Is this related to the issue of permissions?

Reply:
Yes, I do not have the permissions, I had to use my personal account and change the domain to be able to record this test. If you can please record.

Reply:
You can try to provide me with a remote virtual environment (such as VM) where I can log in to my test account, and then you can run the releated test in this environment with my account. This can avoid failures caused by permission problems.
Or I could run these tests for you and send the rerecord yaml file to you~